<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>260</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>StockWidget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/application.qrc">
    <normaloff>:/img/icon-bg.svg</normaloff>:/img/icon-bg.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,16,10,10,8">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_symbol"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_price"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_priceHighLow"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_priceChangePercent"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_difference"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/application.qrc"/>
 </resources>
 <connections/>
</ui>
